### DISCUSSION TITLE
No documentation for init argument

### DISCUSSION THREAD

**Discussion by HeIp** [eslint/eslint#16581] [General] [answered]
So I was following a course to get started with eslint.
And in this course they use the package like so : `eslint --init`
To my surprise the use of this is documented no where.
No mention of using the package like so, why is that ?
Is this an outdated way to use it ? 
I just don't understand why there's no reference to this use case.
If anyone could elaborate on this, I would greatly appreciate it.

**Comment by aladdin-add** [comment, marked as answer]
it has been moved to another repo `@eslint/create-config` in https://github.com/eslint/eslint/pull/15150. as discussed in https://github.com/eslint/rfcs/pull/79

the new way is to use `npm init @eslint/config` as documented, but we keep the `eslint --init` to run it - just for compatibility.
